### PR TITLE
Task details will only show up if task is selected

### LINKED ISSUE #57: Modify UI such that event details are shown only if it is selected
More tasks can be shown without scrolling/filtering

### PR DESCRIPTION
**Pull request by Skaty**
Resolves #57.

### CODE REVIEW

File: src/test/java/guitests/guihandles/TaskCardHandle.java
Comment (advaypal) [inline src/test/java/guitests/guihandles/TaskCardHandle.java]: Could you delete the comments too please? Thanks

File: src/test/java/guitests/guihandles/TaskCardHandle.java
Comment (advaypal) [inline src/test/java/guitests/guihandles/TaskCardHandle.java]: Could you delete the comments here too please? Thanks

**Comment by SukiTsang** [comment]
I think it's good now. But still wait until Advay also approves

**Review by SukiTsang — APPROVED** [review]

**Comment by advaypal** [comment]
@Skaty Looks good to me. Also, could you give me merge access? I would have merged your PR in

**Pull request merged**
